src/commands/init.js

```diff
@@ -61,9 +61,7 @@ class InitCommand extends AbstractAuthenticatedCommand {
 
   async projectSelection() {
     const parsed = this.parse(InitCommand);
-    spinner.pause();
```

Comment:
I remove it from here to be able to have the spinner under error:
![image](https://user-images.githubusercontent.com/48474636/82462157-850cec80-9abb-11ea-9a45-f687c9df4e6f.png)
This is what I had if I leave it here. 
And moving it to the `with-current-project`:
![image](https://user-images.githubusercontent.com/48474636/82462244-a2da5180-9abb-11ea-91af-5532b01b471e.png)